lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -324,6 +326,36 @@ def post(self, request):
             uname = request.POST.get('student_uname', '').strip()
             self.msg = u'<h4>{0}</h4><p>{1}</p><hr />{2}'.format(
                 _('Delete User Results'), self.delete_user(uname), self.msg)
+        elif action == 'rename_user':
+            old_uname = request.POST.get('old_uname', '').strip()
+            new_uname = request.POST.get('new_uname', '').strip()
+            if len(old_uname) == 0 or len(new_uname) == 0:
+                message = 'Usernames cannot be blank'
+            else:
+                try:
+                    rename_user(old_uname, new_uname)
+                except (User.DoesNotExist):
+                    message = 'User \'{old_username}\' does not exist'.format(
+                        old_username=old_uname
+                    )
+                except (IntegrityError):
+                    message = 'User \'{new_username}\' already exists'.format(
+                        new_username=new_uname
+                    )
+                except (PyMongoError):
+                    message = 'Failed to modify username for user \'{old_username}\''.format(
+                        old_username=old_uname
+                    )
+                else:
+                    message = "Changed username of user \'{old_username}\' to \'{new_username}\'".format(
+                        old_username=old_uname,
+                        new_username=new_uname,
+                    )
+            self.msg = u'<h4>{title}</h4><p>{message}</p><hr />{previous_message}'.format(
```

Comment:
Per above note about prefixing variable names, what about `message_previous`? or even `message_old` and `message_new` to follow the same convention as `username_old`/`username_new`?